assets/lang/uz.js

```diff
@@ -0,0 +1,75 @@
+(function ($) {
+$.Redactor.opts.langs['uz'] = {
+	html: 'Kod',
+	video: 'Video',
+	image: 'Rasm',
+	table: 'Jadval',
+	link: 'Havola',
+	link_insert: 'Havolani joylashtirish ...',
+	link_edit: 'Havolani tahrirlash',
+	unlink: 'Havolani o'chirish',
+	formatting: 'Formatlash',
+	paragraph: 'Odatiy matn',
+	quote: 'Iqtibos',
+	code: 'Kod',
+	header1: 'Sarlovha 1',
+	header2: 'Sarlovha 2',
+	header3: 'Sarlovha 3',
+	header4: 'Sarlovha 4',
+	header5: 'Sarlovha 5',
+	bold:  'Qalin',
+	italic: 'Yotiq',
+	fontcolor: 'Matn rangi',
+	backcolor: 'Matn foni',
+	unorderedlist: 'Oddiy ro`yxat',
+	orderedlist: 'Raqamlangan ro`yxat',
+	outdent: 'Shriftni kamaytirish',
+	indent: 'Shriftni oshiring',
+	cancel: 'Bekor qilish',
+	insert: 'Joylash',
+	save: 'Saqlash',
+	_delete: 'O\'chirish',
+	insert_table: 'Jadval joylashtirish',
+	insert_row_above: 'Yuqoridan yangi qator qo`shish',
+	insert_row_below: 'Pastdan yangi qator qo`shish',
+	insert_column_left: 'Chap tomondan ustun qo`shish',
+	insert_column_right: 'O`ng  tomondan ustun qo`shish',
+	delete_column: 'Ustunni o`chirish',
+	delete_row: 'Qatorni o`chirish',
+	delete_table: 'Jadvalni o`chirish',
+	rows: 'Qatorlar',
+	columns: 'Ustunlar',
+	add_head: 'Sarlovha qo\'shish',
+	delete_head: 'Sarlavhani o\'chirish',
+	title: 'Maslahat',
+	image_position: 'Matnni o\'rash',
+	none: 'Yo`q',
+	left: 'Chapga',
+	right: 'O'ng tomonda',
+	image_web_link: 'Rasm havolasi',
+	text: 'Matn',
+	mailto: 'Elektron manzil',
+	web: 'URL',
+	video_html_code: 'Video rolik kodi',
+	file: 'Fayl',
+	upload: 'Yuklash',
+	download: 'Yuklab olish',
+	choose: 'Tanlash',
+	or_choose: 'Yoki tanlang',
+	drop_file_here: 'Faylni bu yerga olib keling',
+	align_left:	'Chap tamon bo`ylab',
+	align_center: 'Markazga',
+	align_right: 'O`ng tamon bo`ylab',
+	align_justify: 'Matnni asoslang',
+	horizontalrule: 'Landshaft o'lchagich',
+	fullscreen: 'To'liq ekran',
+	deleted: 'Chizib tashlandi',
+	anchor: 'Langar',
+	link_new_tab: 'Yangi ichki oynada ochish',
+	underline: 'Belgilangan',
+	alignment: 'Выравнивание',
```

Comment:
Based on all other values, this one doesn't look similar

Reply:
You can check now! Fixed syntax errors